### PR TITLE
[DA-2158] add_fall_survey_fields_in_ps_table

### PR DESCRIPTION
**Pull request by wangy70**
## Resolves *[DA-2158](https://precisionmedicineinitiative.atlassian.net/browse/DA-2158)*
Add Fall Minute Survey columns to participant summary table

## Description of changes/additions
Fall Minute Survey has been launched, add them to participant summary API

## Tests
- [x] unit tests

### CODE REVIEW

**Review by kenny-skaggs — APPROVED** [review]
`questionnaireOnCopeVaccineMinute2` will need to be added to the ppi_questionnaire_fields list in the base-config too.

File: rdr_service/alembic/versions/5bb5c43c1b1f_add_fall_survey_to_ps_table.py [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+
+def downgrade(engine_name):
+    globals()["downgrade_%s" % engine_name]()
+
+
+
+def upgrade_rdr():
+    # ### commands auto generated by Alembic - please adjust! ###
+    op.add_column('participant_summary', sa.Column('questionnaire_on_cope_vaccine_minute_2', rdr_service.model.utils.Enum(QuestionnaireStatus), nullable=True))
+    op.add_column('participant_summary', sa.Column('questionnaire_on_cope_vaccine_minute_2_authored', rdr_service.model.utils.UTCDateTime(), nullable=True))
```
Comment (pegbertsch) [inline rdr_service/alembic/versions/5bb5c43c1b1f_add_fall_survey_to_ps_table.py:41]: Did we stop adding the associated `questionnaire_on_*_time` fields to reflect when the (last) payload was received?  It looks like the summer COPE minute survey was the first one where we only added an `*_authored` field

**Review by pegbertsch — APPROVED** [review]
Approving, but curious about discontinuing the `questionnaire_*_time` fields in `participant_summary'`?

**Comment by wangy70** [comment]
> `questionnaireOnCopeVaccineMinute2` will need to be added to the ppi_questionnaire_fields list in the base-config too.

Yes, will do

File: rdr_service/alembic/versions/5bb5c43c1b1f_add_fall_survey_to_ps_table.py (hunk 1 above)
Comment (wangy70) [reply rdr_service/alembic/versions/5bb5c43c1b1f_add_fall_survey_to_ps_table.py:41]: Yes, I think so.

**Pull request merged**